src/globus_cli/commands/timer/create.py

```diff
@@ -0,0 +1,288 @@
+import datetime
+from typing import List, Optional, Tuple, cast
+
+import click
+from globus_sdk import TimerJob, TransferData
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command, group, mutex_option_group
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+
+from ._common import (
+    DATETIME_FORMATS,
+    JOB_FORMAT_FIELDS,
+    START_HELP,
+    parse_timedelta,
+    read_csv,
+    require_activated_endpoints,
+)
+
+INTERVAL_HELP = """
+    Interval at which the job should run. Use 'w', 'd', 'h', 'm', and 's' as suffixes
+    to specify weeks, days, hours, minutes, and seconds. Examples: '1h 30m', '500s',
+    '24h', '1d 12h', '2w', etc. Must be in order: hours -> minutes -> seconds. You
+    should either use quotes ('1d 2h') or write without spaces (1d2h).
+"""
+
+
+@group("create", short_help="Submit a Timer job", hidden=True)
+def create_command():
+    pass
+
+
+@command("transfer", short_help="Create a recurring transfer job in Timer")
+@click.option(
+    "--name",
+    required=True,
+    type=str,
+    help="Name to identify this job in the timer service (not necessarily unique)",
+)
+@click.option(
+    "--start",
+    required=False,
+    type=click.DateTime(formats=DATETIME_FORMATS),
+    help=START_HELP,
+)
+@click.option(
+    "--interval",
+    required=False,
+    type=str,
+    help=INTERVAL_HELP,
+)
+@click.option(
+    "--source-endpoint",
+    required=True,
+    type=str,
+    help="ID for the source transfer endpoint",
+)
+@click.option(
+    "--dest-endpoint",
+    required=True,
+    type=str,
+    help="ID for the destination transfer endpoint",
+)
```

Comment:
I would like the src and dest to be ENDPOINT_PLUS_OPTPATH items, so that they have the same colon-delimited syntax used throughout the rest of the CLI.
For an example, see: https://github.com/globus/globus-cli/blob/b16ef4c18fcfbfa1c76136209ba76b3223523480/src/globus_cli/commands/transfer.py#L212-L217

This ties in with a note I have below on `--item`.

Additionally, I would strongly prefer that these arguments be positionals, rather than `required=True` options.
We only use required optionals where necessary (or if there's been a mistake), but positional arguments more clearly describe themselves to a user e.g. in usage and synopsis.